composer.json

```diff
@@ -76,7 +76,13 @@
     "laravel": {
       "providers": [
         "A17\\Twill\\TwillServiceProvider"
-      ]
+      ],
+      "aliases": {
```

Comment:
I'm curious to know why the aliases? It seems like the rest of the code is requiring the classes via `use` or referencing fully qualified names. Is this for convenience in Twill projects?